packages/scoutgame/src/topConnector/createTopConnectorEvent.ts

```diff
@@ -0,0 +1,26 @@
+import { prisma } from '@charmverse/core/prisma-client';
+import { getCurrentSeasonStart, getCurrentWeek } from '@packages/dates/utils';
+import { DateTime } from 'luxon';
+
+/**
+ * Create a BuilderEvent of type 'top_connector' for the given builderId
+ */
+export async function createTopConnectorEvent({
+  builderId,
+  referralPoints
+}: {
+  builderId: string;
+  referralPoints: number;
+}) {
+  await prisma.builderEvent.create({
+    data: {
+      bonusPartner: 'top-connector',
```

Comment:
this is not what bonusPartner was intended for. it should only be for real partners that for example live on the partners page